app/src/main/java/io/gnosis/safe/helpers/AddressInputHelper.kt

```diff
@@ -30,37 +28,41 @@ class AddressInputHelper(
     private val dialog =
         BottomSheetDialog(fragment.requireContext()).apply {
             val clipboard = fragment.activity?.getSystemService(Context.CLIPBOARD_SERVICE) as ClipboardManager
+            val binding = BottomSheetAddressInputBinding.inflate(layoutInflater)
+            with(binding) {
+                setContentView(root)
+
+                bottomSheetAddressInputBook.visible(allowAddressBook)
```

Comment:
Should I remove disabled AddressBook support?

Reply:
yes, we don't have ux for it and it might look different in the future. Better to add it when we need it